README.rst

```diff
@@ -19,30 +19,66 @@ If you want to contribute to this documentation set, please consult the `F5 Open
 * Create an `issue <https://github.com/F5Networks/f5-ci-docs/issues>`_ corresponding to the changes/additions you plan to make. When your work is complete, `open a pull request <https://github.com/F5Networks/f5-ci-docs/pulls>`_.
 * Be sure to fetch and rebase often so your fork stays up-to-date!
 
+How to Build, Test, and Deploy Documentation
+````````````````````````````````````````````
 
-Build and Test
-~~~~~~~~~~~~~~
+This project uses CI/CD to build, test, and deploy documentation.
+
+**Tools:**
+
+- `sphinx`: builds HTML, checks syntax, and tests links.
+- `f5-sphinx-theme <https://github.com/f5devcentral/f5-sphinx-theme>`_: F5 theme for sphinx projects.
+- `write-good`: tests grammar.
+- Travis-CI: builds, tests, and deploys documentation.
+- AWS S3/CloudFront: website hosting.
+- Docker image: `f5devcentral/containthedocs <https://hub.docker.com/r/f5devcentral/containthedocs/>`_
+
+**Scripts:**
 
 The `scripts </scripts>`_ directory contains documentation testing resources. We recommend running the test script **before** opening a pull request; if the build associated with your PR doesn't pass, the request won't be accepted.
 
-The test script can be run locally or in a Docker container, which uses an image developed by F5 (https://hub.docker.com/r/f5devcentral/containthedocs/ ). The Ubuntu-based Docker image has all of the dependencies required to build the project documentation pre-installed. If you want to run the test script locally, you'll need to install the `requirements <requirements.txt>`_ first.
+The test script can be run locally or in a Docker container, which uses the F5 ``containthedocs`` image. The Ubuntu-based Docker image has all of the dependencies required to build the project documentation pre-installed. If you want to run the test script locally, you'll need to install the `requirements <requirements.txt>`_ first.
+
+- *docker-docs*: Runs a Docker container mounted to your working directory.
+- *test-docs*: Runs the HTML build, grammar check, and linkcheck.
+
+Building and Testing
+~~~~~~~~~~~~~~~~~~~~
+
+You can use the commands below to build and test your work.
+Commands beginning with `docker` run in a Docker container using the same image used in Travis-CI (`f5devcentral/containthedocs <https://hub.docker.com/r/f5devcentral/containthedocs/>`_).
 
-- scripts/docker-docs.sh -- runs a Docker container with the 'containthedocs' image.
-- scripts/test-docs.sh -- builds the documentation (``make -C docs/ html``); runs ``make linkcheck`` to check internal and external links; and checks grammar with ``write-good``. These tests run in travis-ci and must pass before we can accept a pull request.
++------------------------+--------------------------------------------------------+
+| Command                | Description                                            |
```

Comment:
Are these the commands I want to run in the docker container too?
Does the docker-docs script drop you into the container to run the make targets?

Maybe we should add another statement about viewing the docs, that you'd need to do that from a web browser on your local machine pointed to your local directory. I'm not opposed to hand holding explanations around this.

Reply:
@f5yacobucci -- Yes, docker-docs contains the docker run command that gets you into the container. Also, good point about the hand-holding. 
Do you think it'd be simpler to eliminate the docker-docs script altogether and just provide the docker run command in the makefile?

Reply:
I think leaving the script is fine, but just add a little more explanation around building from the container and then viewing from the local desktop.